### PR TITLE
feat: i18n for schedules

### PR DESCRIPTION
**Pull request by WolfgangAl**
This adds i18n/l10n options for the following:

- the schedule summaries shown in the event creation dialog (by overriding dayspan/Schedule.prototype.describe + describeRule + describeArray and dayspan/DaySpan.prototype.summary)
- the pattern labels in the frequency dropdown 
- rank numbers (by overriding dayspan/Suffix.CACHE use)

with default values for English

### CODE REVIEW

**Comment by WolfgangAl** [comment]
*NB*: there is some overlap between this PR and #112

**Comment by ClickerMonkey** [comment]
Thanks for the work here - but this adds the moment dependency back to `dayspan-vuetify` - while I'm working on removing it from `dayspan`! (so there should be no moment dependency soon, making this code not necessary).

This does create a great blueprint though on the code in `dayspan` which affects the localizations in `dayspan-vuetify` - so thank you for that!

**Comment by WolfgangAl** [comment]
Of course, you are correct. This reintroduces or at least further affirms the dependency on Moment. I just didn’t think adding i18n to dayspan itself was worth the effort :)

The problem is, Moment/DayJS and other libraries provide excellent date/time functionality and localisation options. It would be a shame to have to copy that into dayspan/dayspan-vuetify just to remove a dependency most implementations will probably add in anyway. Forking dayspan/dayspan-vuetify into separate Moment/DayJS/… versions doesn’t seem prudent either. So that leaves dependency injection or plugins/mixins. We can’t count on the API of date/time libraries being sufficiently compatible to make dependency injection work.

Basically we’re left with plugins. That would mean providing at least Moment and DayJS plugins out-of-the-box and probably allowing for a roll-your-own scenario as well. Those plugins will need i18n/l10n options as well because formatting options will undoubtedly be different between them (even though DayJS is mostly compatible with Moment it does not support the ‘L’ Moment formats for example).

So what are your thoughts about this? What direction are you planning on taking?

**Comment by ClickerMonkey** [comment]
I know it's been awhile, but I've been at work on this - I've added i18n support to dayspan and am working on removing the dependency of Moment. Hopefully sometime within the next week it'll be done and ready.
